temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/AmountModal.kt

```diff
@@ -317,6 +319,36 @@ fun AmountKeyboard(
     FourthRowExtra: (@Composable RowScope.() -> Unit)? = null,
     onBackspace: () -> Unit,
 ) {
+    /**
+     * Retrieve `features` via EntryPointAccessors. `isStandardLayout` is stable and
+     * only changes via settings, so there isn't unnecessary recompositions.
+     * `isStandardLayout` doesn't change while the keyboard is shown.
+     * `AmountKeyboard` is self-contained, making it easy to reuse without passing parameters.
+     * Layout changes are rare, so recomposition has minimal impact on performance.
+     **/
+
+    val context = LocalContext.current
+    val features = EntryPointAccessors.fromApplication(
```

Comment:
this must be `remember {}`

Reply:
This would also break previews and screenshot tests but such doesn't seem to exist

Reply:
In my testing I didn't notice it breaking previews, and yes my bad it should be `remember`